airgun/views/containerimagetag.py

```diff
@@ -36,12 +36,31 @@ def is_displayed(self):
     @View.nested
     class details(SatTab):
         product = ReadOnlyEntry(name='Product')
-        repository = ReadOnlyEntry(name='Repository')
+        schema = ReadOnlyEntry(name='Schema Version')
+        manifest_type = ReadOnlyEntry(name='Manifest Type')
+        digest = ReadOnlyEntry(name='Digest')
 
     @View.nested
     class lce(SatTab):
         TAB_NAME = 'Lifecycle Environments'
         table = SatTable(
             './/table',
-            column_widgets={'Environment': Text('./a'), 'Content View Version': Text('./a')},
+            column_widgets={
+                'Environment': Text('./a'),
```

Comment:
Please take a look into this reference PR "[fix bad table row locator](https://github.com/SatelliteQE/airgun/commit/1ddd785983d534c7ef3e370ce3f53e53badfc315)" raised by @sambible 

`'Environment': Text('.//a')`

Reply:
Although both options work, after off-line communication we decided to keep the current version.